src/types.cpp

```diff
@@ -23,12 +23,18 @@ std::ostream &operator<<(std::ostream &os, const SizedType &type)
   }
   if (type.is_pointer)
     os << "*";
+  if (type.type == Type::integer && (type.size != 8 || type.is_signed)) {
+    os << "(";
+    if (type.is_signed)
```

Comment:
Shouldn't it be

```suggestion
    if (!type.is_signed)
```

?